src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/components/dot-layout-designer/dot-layout-designer.component.html

```diff
@@ -1,21 +1,21 @@
 <div class="dot-layout-designer" [formGroup]="group">
     <div class="dot-layout-designer__header" *ngIf="group.get('header').value">HEADER</div>
     <div class="dot-layout-designer__main">
-        <div class="dot-layout-designer__sidebar--left" [ngClass]="'dot-layout-designer__sidebar--' + group.get('sidebar.width').value"
-            *ngIf="group.get('sidebar.location').value === 'left'">
-            SIDEBAR
-            <dot-sidebar-properties [group]="group.get('sidebar')" class="dot-layout-designer__sidebar-properties"></dot-sidebar-properties>
+        <div class="dot-layout-designer__sidebar--left" [ngClass]="'dot-layout-designer__sidebar--' + group.get('sidebar').value['width']"
+            *ngIf="group.get('sidebar').value['location'] === 'left'">
+            <dot-sidebar-properties formControlName="sidebar" class="dot-layout-designer__sidebar-properties"></dot-sidebar-properties>
+            <dot-edit-layout-sidebar formControlName="sidebar"></dot-edit-layout-sidebar>
```

Comment:
Why here we have dot-sidebar-properties and dot-edit-layout-sidebar?, it should not be just dot-edit-layout-sidebar

Reply:
Yup... I think we should move ```dot-sidebar-properties``` inside ```dot-edit-layout-sidebar```

Reply:
And the <div> that wrap it too.